src/app/Events/EventTypes.tsx

```diff
@@ -203,7 +203,7 @@ export const EventTypes: React.FunctionComponent<EventTypesProps> = (props) => {
   );
 
   const onToggle = React.useCallback(
-    (t: EventType, index: number) => {
+    (t: EventType, _index: number) => {
```

Comment:
If `_index` is not use, we can just remove it i think. Might need to update where it is being called.